source/run-command/tests/unified/run-command.yml

```diff
@@ -0,0 +1,242 @@
+                $db: dbWithRC
+              commandName: ping
+
+  - description: executes a ping without attaching writeConcern
+    operations:
+      - name: runCommand
+        object: dbWithWC
+        arguments:
+          commandName: ping
+          command:
+            ping: 1
+        expectResult:
+          ok: 1
+    expectEvents:
+      - client: client
+        events:
+        - commandStartedEvent:
+              command:
+                ping: 1
+                writeConcern: { $$exists: false }
+                $db: dbWithWC
+              commandName: ping
+
+  - description: executes a ping and does not retry retryable error
+    operations:
+      - name: failPoint
+        object: testRunner
+        arguments:
+          client: client
+          failPoint:
+            configureFailPoint: failCommand
+            mode: { times: 1 }
+            data:
+              failCommands: [ping]
+              closeConnection: true
+      - name: runCommand
+        object: plainDb
+        arguments:
+          commandName: ping
+          command:
+            ping: 1
+        expectError:
+          isError: true
+
+  - description: executes a insert via runCommand inside a transaction
+    runOnRequirements:
+      - topologies: [ replicaset, sharded-replicaset, load-balanced, sharded ]
+        minServerVersion: '4.4'
+    operations:
+      - name: withTransaction
+        object: session
+        arguments:
+          callback:
+            - name: runCommand
+              object: plainDb
+              arguments:
+                session: session
+                commandName: insert
+                command:
+                  insert: collection
+                  documents: [ { _id: 1 } ]
+                  ordered: true
+              expectResult: { $$unsetOrMatches: { insertedId: { $$unsetOrMatches: 1 } } }
+    expectEvents:
+      - client: client
+        events:
+          - commandStartedEvent:
+              command:
+                insert: collection
+                documents: [ { _id: 1 } ]
+                ordered: true
+                lsid: { $$sessionLsid: session }
+                txnNumber: 1
+                startTransaction: true
+                autocommit: false
+                # omitted fields
+                readConcern: { $$exists: false }
+                writeConcern: { $$exists: false }
+              commandName: insert
+              databaseName: plainDb
+          - commandStartedEvent:
+              command:
+                commitTransaction: 1
+                lsid: { $$sessionLsid: session }
+                txnNumber: 1
+                autocommit: false
+                # omitted fields
+                writeConcern: { $$exists: false }
+                readConcern: { $$exists: false }
+              commandName: commitTransaction
+              databaseName: admin
+
+  - description: executes a ping with stableApi fields attached
```

Comment:
this should be something like "attaches stableApi fields to type of command when configured on the client"